src/services/query.ts

```diff
@@ -197,5 +198,41 @@ export function sanitizeStreamSelector(expression: string) {
   return expression.replace(/\s*,\s*}/, '}');
 }
 
+/**
+ * Variables that contain other variables are not interpolated, until interpolate is called again.
+ */
+export function interpolateExpression(sceneObject: SceneObject, uninterpolatedExpr: string | undefined) {
+  let expr = sceneGraph.interpolate(sceneObject, uninterpolatedExpr);
+
+  // Need to interpolate expressions with VAR_JSON_FIELDS_EXPR twice
```

Comment:
Why?

Reply:
Looks like the scenes interpolate method doesn't interpolate nested variables, `jsonFields` is nested within the `FORMAT_EXPR` variables in variables.ts

Reply:
Updated the comment: https://github.com/grafana/logs-drilldown/pull/1122/commits/1e836d17bf949b489d3e327f67809982ede21aca